src/components/DisplayArticles.jsx

```diff
@@ -37,7 +37,7 @@ const DisplayArticles = () => {
               to={`/articles/${article.id}`}
               data-cy={"article-" + article.id}
             >
-              <Image src="https://slack-imgs.com/?c=1&o1=ro&url=https%3A%2F%2Fwww.grade.com%2Fwp-content%2Fuploads%2F2020%2F03%2Fplaceholder.png" />
+              <Image data-cy="image" src={article.url} />
```

Comment:
Add a conditional so if the article doesn't have an image attached the image element should not exist . Otherwise it will be a broken image there

Reply:
fixed by connor